src/go/cmd/update-collection-v3/migrations/tracing-config/migrate.go

```diff
@@ -0,0 +1,50 @@
+package tracingconfig
+
+import (
+	"bytes"
+	"fmt"
+
+	"gopkg.in/yaml.v3"
+)
+
+func Migrate(yamlV2 string) (yamlV3 string, err error) {
+	valuesV2, err := parseValues(yamlV2)
```

Comment:
Now, we use the style in which we use `input/output` instead of `V2/V3`
some comments from other pull requests:
https://github.com/SumoLogic/sumologic-kubernetes-tools/pull/394#discussion_r1034522229
https://github.com/SumoLogic/sumologic-kubernetes-tools/pull/398

Reply:
🤷‍♀️

Reply:
I suggest to use new style in new files with code.

Reply:
Thanks Kasia, I've applied all the changes related to structs naming in every `tracing-*` migration.